client/src/redux/settings/update-legacy-certificate-saga.js

```diff
@@ -17,7 +17,7 @@ function* updateLegacyCertSaga({
   };
   // shape to update completed challenges in redux store
   let reduxShape = [];
-  for (let obj in challengesToUpdate) {
+  for (var obj in challengesToUpdate) {
```

Comment:
Just a question.  Why change this to `var`?

Reply:
linter was shouting at me but I will revert this.
Thanks for the review @RandellDawson